### DISCUSSION TITLE
in the FDS (Fire Dynamics Simulator) simulation, the WARNING message "WARNING: Minimum density, 0.016 kg/m³, clipped in Mesh 1" is encountered.

### DISCUSSION THREAD

**Discussion by gdut21122** [firemodels/fds#12682] [Q&A]
![de52dffb3c04d1d9b754e75ac34a28c](https://github.com/firemodels/fds/assets/164824542/6d3b3637-4b01-43a8-ba7e-a73306049521)

**Comment by Er9y714** [comment]
See #12676

**Comment by mcgratta** [comment]
If this is a new issue that you want someone to look at, provide details and a simplified input file on the Issue Tracker. I do not understand what the question is here.
